ui/src/types/variables.ts

```diff
@@ -0,0 +1,7 @@
+export {VariableProperties} from 'src/client'
+import {Variable as GenVariable} from 'src/client'
+import {Label} from 'src/types'
+
+export interface Variable extends GenVariable {
+  labels?: Label[]
```

Comment:
I think we are ensuring that labels exists as a param even though it may be empty by adding defaults?

Reply:
But that doesn't necessarily mean it'll be guaranteed to return from the BE with labels

Reply:
from BE -> add defaults function -> labels are guaranteed and Variable describes the type after defaults have been added.